### PR TITLE
Nikosai/refactoring

### PR DESCRIPTION
**Pull request by nikosai**
### Renamed slack-connection to slack
The types (in settings.json) "slack-channel" and "slack-connection" are combined into "slack".

### Updated README
Both Japanese and English READMEs are now updated.

### CODE REVIEW

**Comment by hikalium** [comment]
We need to add default values for parameters.

**Comment by nikosai** [comment]
I know proton's "pro" means "professional of English".

**Comment by hikalium** [comment]
@proton-yoko Please review english part of README.

**Comment by nikosai** [comment]
@proton-yoko How is the progress?

File: README.en.md [hunk 1]
```diff
@@ -80,56 +84,81 @@ Example:
...
+#### interfaces : any[] 
+* settings about devices
+* type: "stdio"
+	+ standard input & output. No parameters to set.
+* type: "slack"
+	+ team : string
+		- name of your team `TEAMNAME.slack.com`
+	+ token : string
+		- API Token (Available: https://api.slack.com/bot-users)
+		- After you get token, you should add bot in slack>Apps, and then invite the bot to teams.
```
Comment (proton-yoko) [inline README.en.md]: After you get a token, you should add a bot in slack>Apps, and then invite the bot to teams.

File: README.en.md [hunk 2]
```diff
@@ -80,56 +84,81 @@ Example:
...
+	+ output : boolean
+		- whether this driver is used to output(if false, replies are displayed on stdout, instead of sending to slack)
```
Comment (proton-yoko) [inline README.en.md]: used "to" output sounds a bit strange

**Review by proton-yoko — CHANGES_REQUESTED** [review]
please change it! (if you have the motivation)

**Review by proton-yoko — APPROVED** [review]
ack

**Pull request merged**